frontend/src/App.js

```diff
@@ -10,6 +10,7 @@ import Video from "./Pages/Video";
 import Loader from "./Components/Loader";
 import Upload from "./Pages/Upload";
 import Channel from "./Pages/Channel";
```

Comment:
Suggestion: Can't these Components be imported from an index file from their respective folders?

Reply:
It can, will do all those in last.